### DISCUSSION TITLE
The LargeChange of vertical scrollbar of DataGrid is not set, is this by design or missing?

### DISCUSSION THREAD

**Discussion by JerryJian** [AvaloniaUI/Avalonia#16512] [Q&A] [answered]
According to the source code: https://github.com/AvaloniaUI/Avalonia/blob/ff13f06b9a5f69479c4267e67f0b4bae27b0a04b/src/Avalonia.Controls.DataGrid/DataGrid.cs#L5840

The LargeChange of vertical scrollbar is not set compared to the horizontal scrollbar. Which causes when click the blank area, the data in the DataGrid is not scroll by page.

Is this by design or missing, thanks.

**Comment by maxkatz6** [comment, marked as answer]
Seems like it was missed in the initial port. Original one has it:
https://github.com/microsoftarchive/SilverlightToolkit/blob/8dd19e0c12a8059270d244f261375b30d6aec26b/Release/Silverlight3/Source/System.Windows.Controls.Data/DataGrid/DataGrid.xaml.cs#L6438-L6439

Probably Avalonia didn't have LargeChange property back then.
